### PR TITLE
[FIX] Services with empty description

### PR DESCRIPTION
**Pull request by mileo**
Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

### CODE REVIEW

**Comment by mileo** [comment]
Avoid errors:

```
    return f(dbname, *args, **kwargs)
  File "/home/mileo/Projects/oca12/parts/odoo/odoo/http.py", line 339, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/mileo/Projects/oca12/parts/odoo/odoo/http.py", line 941, in __call__
    return self.method(*args, **kw)
  File "/home/mileo/Projects/oca12/parts/odoo/odoo/http.py", line 519, in response_wrap
    response = f(*args, **kw)
  File "/home/mileo/Projects/oca12/parts/rest-framework/base_rest/controllers/api_docs.py", line 38, in api
    return self.make_json_response(service.to_openapi())
  File "/home/mileo/Projects/oca12/parts/rest-framework/base_rest/components/service.py", line 224, in to_openapi
    root["info"] = self._get_openapi_info()
  File "/home/mileo/Projects/oca12/parts/rest-framework/base_rest/components/service.py", line 232, in _get_openapi_info
    "description": textwrap.dedent(getattr(self, "_description", "")),
  File "/usr/lib/python3.8/textwrap.py", line 430, in dedent
    text = _whitespace_only_re.sub('', text)
TypeError: expected string or bytes-like object - - -

```
image

File: shopinvader/services/address.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class AddressService(Component):
     _name = "shopinvader.address.service"
     _usage = "addresses"
     _expose_model = "res.partner"
+    _description = "Service providing Address Service"
```
Comment (simahawk) [inline shopinvader/services/address.py]: I suggest "Shopinvader XXX Service"

File: shopinvader_lead/services/lead.py [hunk 2]
```diff
@@ -12,6 +12,8 @@ class LeadService(Component):
     _name = "shopinvader.lead.service"
     _usage = "lead"
     _expose_model = "crm.lead"
+    _description = """Service for create leads from your website,
```
Comment (simahawk) [inline shopinvader_lead/services/lead.py]: "Shopinvader CRM lead service"
Your long description should go into the class docstring IMO.

**Comment by rousseldenis** [comment]
@mileo Could you attend comments and rebase this ?

**Comment by mileo** [comment]
> @mileo Could you attend comments and rebase this ?

@rousseldenis done!

**Review by rousseldenis — APPROVED** [review]

**Review by bealdav — APPROVED** [review]

**Comment by rousseldenis** [comment]
@simahawk Could you update your review ?

File: shopinvader/services/invoice.py [hunk 3]
```diff
@@ -13,7 +13,9 @@ class InvoiceService(Component):
     _name = "shopinvader.invoice.service"
     _usage = "invoice"
     _expose_model = "account.invoice"
-    _description = "Service providing a method to download invoices"
+    _description = (
```
Comment (simahawk) [inline shopinvader/services/invoice.py:16]: this goes to the docstring

**Review by simahawk — APPROVED** [review]

**Pull request merged**